airflow/providers/docker/operators/docker.py

```diff
@@ -30,7 +30,7 @@
 
 from docker.constants import DEFAULT_TIMEOUT_SECONDS
 from docker.errors import APIError
-from docker.types import LogConfig, Mount
+from docker.types import LogConfig, Mount, Ulimit
```

Comment:
Could you move it to `if TYPE_CHECKING` block?

Reply:
Will do, also will fix the failing tests I missed

Reply:
Hmm, ran the tests locally and static checks passed for me without `if TYPE_CHECKING`. Is it still necessary?

Reply:
If the static checks pass, no need for it, I will check it once the PR is merged

Reply:
Now spell checking is failing, so I'm adding `ulimit` to `spelling_wordlist.txt`.